starport/pkg/chaincmd/chaincmd.go

```diff
@@ -120,7 +122,12 @@ func (c ChainCmd) AddKeyCommand(accountName string) step.Option {
 		constJSON,
 	}
 	command = c.attachKeyringBackend(command)
-	return step.Exec(c.appCmd, c.attachHome(command)...)
+
+	// Check version
+	if c.isStargate() {
```

Comment:
How about simplifying this and other similar like below:

```go
return step.Exec(c.daemonCmd(), c.attachCLIHome(command)...)

// where daemonCmd is:
func (c ChainCmd) daemonCmd() string {
    if c.isStargate() {
        return c.appCmd
    }
    return c.cliCmd
}
```

Reply:
`c.attachHome(command)` is changed to `c.attachCLIHome(command)` as well so we should have

```
func (c ChainCmd) daemonCmd(command) string {
    if c.isStargate() {
        return step.Exec(c.appCmd, c.attachHome(command)...)
    }
    returnstep.Exec(c. cliCmd, c. attachCLIHome(command)...)
}
```

The name `daemonCmd ` is a bit confusing because Launchpad daemon is also `appd`

I would rather call it `cliCmd` in the sense that for launchpad cli is explicitly `appcli` and the cli for Stargate is `appd`